### ISSUE TITLE
ReferenceError: replace is not defined

### ISSUE THREAD

**Issue by YooJeongWoo** [froala/wysiwyg-editor#4055]
##### Behavior.
Pasting images from a clipboard (cmd +c, cmd + v) cause `Uncaught ReferenceError: replace is not defined` error.
Seems like there's not-defined function called replace inside 'onPasteImageUplaod'  hook.

```
Uncaught ReferenceError: replace is not defined
at Ze (plugins.pkgd.min.js:7)
at plugins.pkgd.min.js:7
Ze @ plugins.pkgd.min.js:7
(anonymous) @ plugins.pkgd.min.js:7
setTimeout (async)
a.onload @ plugins.pkgd.min.js:7
load (async)
(anonymous) @ plugins.pkgd.min.js:7
each @ froala_editor.min.js:7
et @ plugins.pkgd.min.js:7
m @ froala_editor.min.js:7
M @ froala_editor.min.js:7
o @ froala_editor.min.js:7
d @ froala_editor.min.js:7
t @ froala_editor.min.js:7
```

##### Steps to reproduce the problem.
Paste any image from a clipboard.

##### Editor version.
3.2.2

##### Browser.
Chrome

**Comment by Rainbowm** [comment]
i have the same error

**Comment by a971106a** [comment]
same error

**Comment by AlexeyShaykov** [comment]
same

**Comment by jkhaoqi110** [comment]
same error

**Comment by earshinov** [comment]
Said to be fixed in v3.2.4: https://froala.com/wysiwyg-editor/changelog/#3.2.4
